src/core/CoreTransaction.jl

```diff
@@ -1,145 +1,72 @@
 # This file is a part of GraknClient.  License is MIT: https://github.com/Humans-of-Julia/GraknClient.jl/blob/main/LICENSE
 
-mutable struct CoreTransaction <: AbstractCoreTransaction
-    type::Union{Nothing,Int}
+struct CoreTransaction <: AbstractCoreTransaction
+    type::Union{Nothing,Int32}
```

Comment:
Define `Maybe{type}` and use it instead of having all of these `Union{Nothing, type}`

Reply:
Done.